blazor/file-manager/how-to/add-custom-tool-bar.md

```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Adding Custom Item to Toolbar in Blazor FileManager Component
 
-The toolbar items can be customized using the `ToolbarSettings` API and `ToolbarItemClicked` events.
+The toolbar items can be customized using the `ToolbarSettings` API and `ToolbarCreated` events.
 
-The following example shows adding a custom item in the toolbar. The new toolbar button is added using `ToolbarSettings`. The event is used to add an `ToolbarItemClicked` event handler to the new toolbar button.
+The following example shows adding a custom item in the toolbar. The new toolbar button is added using `ToolbarSettings`. The `ToolbarCreated` event is used to add extra properties for the new toolbar button.
```

Comment:
The new toolbar button 'Custom' is added using `ToolbarSettings` property. Also, added prefix icon and tooltip to the newly added toolbar button using `ToolbarCreated` event.